corehq/apps/hqwebapp/templates/hqwebapp/base.html

```diff
@@ -53,7 +53,10 @@
       <link rel="apple-touch-icon" sizes="180x180" href="{% static 'hqstyle/images/commcare-icon-180x180.png' %}"/>
     {% endif %}
 
-    {% block stylesheets %}{% endblock stylesheets %}
+    {% block stylesheets %}
```

Comment:
Hey, I think my earlier comment misdirected you. Rather than change this file, in order to make the change only affect the info basic page, you'd add code like this to `info_basic.html`:
```
{% block stylesheets %}{{ block.super }}
   <link rel="stylesheet" href="{% static '@mapbox/mapbox-gl-geocoder/dist/mapbox-gl-geocoder.css' %}" type="text/css" />
{% endblock stylesheets %}
```